email_templates/suppliers_new_briefs_questions_answers.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+</head>
+<body>
+Dear supplier,
+<br /><br />
+In the last day, buyers answered questions about:
```

Comment:
I just talked with @tracy-content about this and she said that the comma is fine. @TheDoubleK and @idavidmcdonald could you confirm if this is ready for merging, then? Or some more stuff to address?